client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -16,12 +17,16 @@ import Masterbar from 'layout/masterbar/masterbar';
 import './style.scss';
 
 export default function() {
+	const translate = useTranslate();
+
 	return (
 		<Masterbar>
 			<Item
 				className="masterbar__item-home"
 				url="/"
-				tooltip="Jetpack Cloud Homepage" /* @todo: localize the string */
+				tooltip={ translate( 'Jetpack Cloud Dashboard', {
+					comment: 'Jetpack Cloud masterbar nav item',
```

Comment:
A nuanced nit here: sometimes translators may not know what a masterbar is (as it's an internal term). We might want to change it to "top navigation bar" or something.